mscore/musescore.h

```diff
@@ -742,6 +742,8 @@ class MuseScore : public QMainWindow, public MuseScoreCore {
       bool exportPartsPdfsToJSON(const QString& inFilePath, const QString& outFilePath = "/dev/stdout");
       /////////////////////////////////////////////////
 
+      void scoreUnrolled(MasterScore * original);
```

Comment:
Concerning the style, `Type* somename` is usually used in MuseScore:
https://github.com/musescore/MuseScore/blob/0c4aefdd6c5bf49eb7b92c36ffa7bd858296174f/mscore/musescore.h#L453